zipkin-lens/jest.config.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2015-2019 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+module.exports = {
+  setupFilesAfterEnv: ['./src/setup-test.js'],
```

Comment:
`setupTestFrameworkScriptFile` is deprecated.
We should use `setupFilesAfterEnv` instead of it.

https://jestjs.io/docs/en/configuration#setupfilesafterenv-array